mw_url_rewrite_test.go

```diff
@@ -560,6 +560,52 @@ func TestRewriterTriggers(t *testing.T) {
 				r,
 			}
 		},
+		// func() TestDef {
```

Comment:
@ilijabojanovic I think there's some weird caching behaviour here, if I uncomment this test it fails even though the rewrite logic is triggered.

The rewrite path prints as "/change/to/me/$tyk_context.trigger-0-payload" which comes from the previous test, and the URL stays as "/change/to/me/ignore"

Any thoughts?